pkg/kubectl/cmd/expose.go

```diff
@@ -110,7 +111,7 @@ func RunExpose(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 		return err
 	}
 
-	mapper, typer := f.Object()
+	mapper, typer := f.Object(cmdutil.GetIncludeThirdPartyAPIs(cmd))
```

Comment:
The use-case for `true` here is that someone has gone to the trouble of creating a customized generator for a third party resource that they're pass through to expose?  Does that work without re-compiling (I haven't tried)?  I don't know that I see the need